wesnoth-units.po

```diff
@@ -1861,11 +1886,10 @@ msgstr "Pouštní podpalovačka"
 
 #. [unit_type]: id=Dune Harrier, race=dunefolk
 #: data/core/units/dunefolk/Harrier.cfg:8
-#, fuzzy
-#| msgid "Drake Warrior"
 msgid "female^Dune Harrier"
-msgstr "Drakonidský válečník"
+msgstr "Pouštní běžkyně"
```

Comment:
Nikdy jsem pouštní lid nehrál, tak nevím, o co jde, ale Google Translator mi nabídl význam "a person who engages in persistent attacks on others or incursions into their land", což mi přijde něco jako nájezdník/narušitel. Tak doufám, že máte v té běžkyni pevnou jistotu, já to zatím víc nezkoumal.

Reply:
Já jsem se držel té možnosti, že je to ten co štve zvěř. Navíc je to nejspíš další stupeň chodkyně, takže mi to k tomu šlo. Ale nejradši bych k obojímu dal ještě něco lepšího, ale nějak jsem neměl inspiraci.

Reply:
Právě, že už trochu zápasím s tím, že překládám jednotky, co jsem nikdy nehrál. Tyto nebyly ani v původní inspiraci (Kalifech), ani nejsou v současné stable verzi.
Právě, že ani nevím, jestli vůbec jsou nějací mužští pouštní chodci a běžci. A jestli jsou to vůbec pěší jednotky.
Možná bych to zatím nechal a až vyjde nová stable, tak to doleštíme.

Reply:
je to na vás

já se zatím snažím překládat jen to, o čem si myslím, že tomu rozumím